components/Archetype/basic_usage.rst

```diff
@@ -0,0 +1,29 @@
+Basic Usage
+===========
+
+.. code-block:: php
+
+    // File example: src/script.php
+    <?php
+
+    // update this to the path to the "vendor/" directory, relative to this file
+    require_once __DIR__ . '/vendor/autoload.php';
+
+    use Sylius\Component\Archetype\Model\Archetype;
+    use Sylius\Component\Product\Model\Product;
```

Comment:
``ClassThatImplementsArchetypeSubjectInterface`` That is not clear. I'd use a real world example, maybe just something different than Product and add a comment about implementing that interface.